components/AddForm/__snapshots__/AddForm.spec.jsx.snap

```diff
@@ -3,41 +3,127 @@
 exports[`AddForm component should render children forms 1`] = `
 <DocumentFragment>
   <div
-    class="govuk-grid-row"
+    class="govuk-form-group"
   >
     <div
-      class="govuk-form-group govuk-grid-column-one-half"
+      class="govuk-form-group"
     >
-      <label
-        class="govuk-label govuk-label--m"
-        for="category"
+      <div
+        class="govuk-form-group"
       >
-        Choose a form category 
-      </label>
-      <select
-        class="govuk-select width-override-30"
-        data-testid="category"
-        id="category"
-        name="category"
-        style="width: 100%;"
-      >
-        <option
-          value=""
+        <label
+          class="govuk-label govuk-label--m"
+          for="formList"
         >
-          Choose one
-        </option>
-        <option
-          value="General"
+          Choose a form 
+        </label>
+        <div
+          aria-expanded="true"
+          aria-haspopup="listbox"
+          aria-labelledby="downshift-1-label"
+          aria-owns="downshift-1-menu"
+          class="inputContainer"
+          role="combobox"
         >
-          General
-        </option>
-        <option
-          value="FAST"
+          <input
+            aria-autocomplete="list"
+            aria-controls="downshift-1-menu"
+            aria-labelledby="downshift-1-label"
+            autocomplete="off"
+            class="input"
+            data-testid="formList"
+            id="formList"
+            name="formList.autocomplete"
+            placeholder="Select or search for a value"
+            style="border-color: #000;"
+            value=""
+          />
+        </div>
+        <ul
+          aria-labelledby="downshift-1-label"
+          class="suggestionList"
+          id="downshift-1-menu"
+          role="listbox"
         >
-          FAST
-        </option>
-      </select>
+          <li
```

Comment:
we should mock the `data/googleForm` import used by this snapshot, otherwise if we are going to change that list this test is going to break